hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -188,6 +188,7 @@ public Void call() throws Exception {
 
   public void setConfiguration(OzoneConfiguration configuration) {
     this.conf = configuration;
+    this.secConf = new SecurityConfig(conf);
```

Comment:
I don't think we need this initialization, the secConf variable is initialized in the start() method as well, and that is in the code path for the startup.